src/components/forms/DateField/DateField.js

```diff
@@ -1,38 +1,33 @@
-import React, { useState } from 'react';
-import FormControl from '@material-ui/core/FormControl';
+import React from 'react';
+import { Field } from 'formik';
 import { KeyboardDatePicker } from '@material-ui/pickers';
+import { fieldToKeyboardDatePicker } from 'formik-material-ui-pickers';
+import FormControl from 'components/forms/FormControl';
 
-import useStyles from './styles';
-import { useForm } from 'components/FormProvider';
+function FormikKeyboardDatePicker({ children, onChange, ...props }) {
+  const datePickerProps = fieldToKeyboardDatePicker(props);
+  if (onChange) datePickerProps.onChange = onChange;
```

Comment:
No change here, but I imagine other form components might evolve to need this if they require greater control at some point?

Reply:
Yes! I've had to implement this in more fields in my next PR. Stay tuned. ;)